openbas-front/src/admin/components/components/teams/TeamPopover.tsx

```diff
@@ -5,26 +5,26 @@ import * as React from 'react';
 
 import type { ExercisesHelper } from '../../../../actions/exercises/exercise-helper';
 import type { OrganizationHelper, TagHelper } from '../../../../actions/helper';
-import type { TeamInputForm, TeamStore } from '../../../../actions/teams/Team';
+import { TeamInputForm } from '../../../../actions/teams/Team';
 import { deleteTeam, updateTeam } from '../../../../actions/teams/team-actions';
 import type { TeamsHelper } from '../../../../actions/teams/team-helper';
 import Dialog from '../../../../components/common/Dialog';
 import Transition from '../../../../components/common/Transition';
 import { useFormatter } from '../../../../components/i18n';
 import { useHelper } from '../../../../store';
-import type { TeamOutput, TeamUpdateInput } from '../../../../utils/api-types';
+import type { Team, TeamOutput, TeamUpdateInput } from '../../../../utils/api-types';
 import { useAppDispatch } from '../../../../utils/hooks';
 import { Option, organizationOption, tagOptions } from '../../../../utils/Option';
 import { TeamContext } from '../../common/Context';
 import TeamForm from './TeamForm';
 
 interface TeamPopoverProps {
-  team: TeamStore | TeamOutput;
+  team: Team | TeamOutput;
```

Comment:
Wouldn't it make more sense to have team only be a TeamOutput ? Both types overlap and I feel it would prevent issues down the line.